create-operator-bundle.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+set -x
+
+#setting sudo
+sudo echo "setting sudo root"
+
+# Initialization
+DEFAULT_CONTAINER_EXECUTABLE="docker"
+CONTAINER_CLIENT="${CONTAINER_EXECUTABLE:-$DEFAULT_CONTAINER_EXECUTABLE}"
+HOSTNAME=cnftest-local.redhat.com
```

Comment:
where did you get this hostname value from ?

Reply:
This is just a random name